lib/observer/observer.ex

```diff
@@ -0,0 +1,14 @@
+defmodule Observer do
+require Satellite.Constants
+  defstruct [:latitude, :longitude, :height]
+
+  def create_from(latitude_degrees, longitude_degrees, height_km) do
```

Comment:
helper function to make deg to radian conversion less error prone (its easy to forget to do this in the code)